src/include/miopen/problem_description.hpp

```diff
@@ -38,6 +38,8 @@
 #include <cstdint>
 #include <string>
 
+#define FIN_OLD_PROBLEM_DESCRIPTION_COMPAT 1
```

Comment:
Is this flag documented somewhere?

Reply:
No, it is supposed to be used with MIFin for backward compatibility and will be removed soon.